content/docs/command-reference/remote/add.md

```diff
@@ -47,13 +47,6 @@ DVC will determine the [type of remote](#supported-storage-types) based on the
 > include them all. The command should look like this: `pip install "dvc[s3]"`.
 > (This example installs `boto3` library along with DVC to support S3 storage.)
 
-DVC supports the concept of a _default remote_. For the commands that accept a
```

Comment:
It looks quite important, no?

Reply:
Hmmm yeah I didn't mean to just remove this. I thought it was rewritten rather. Let me check, maybe a bad stash/unstash operation on my side... ⌛